ExcelExample-FrenchTraffic/main.py

```diff
@@ -0,0 +1,147 @@
+import os
+import sys
+import math
+import json
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+gDEBUG = False
+gGRAPH = False
+gSAVE = True
+
+gPATHI = ""
+gPATHO = ""
+
+isDAFNI = os.environ.get("ISDAFNI")
+print("ISDAFNI Environment variable = ", isDAFNI, type(isDAFNI))
+if os.environ.get("ISDAFNI") == "True":
```

Comment:
why not use the isDAFNI variable here?

also you can shorten this by using `os.getenv("ISDAFNI")` and optionally provide a second arg to be the default value e.g.: `os.getenv("ISDAFNI", False)`